server/static/devError.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
This and `prodError.html` have been borrowed as it is from `koa-onerror` to tone down the overall scope of the PR. It'd be good if both match Outline's theme but I'm also in favor of doing that separately.

Reply:
Okay, reasonable. Lets call these `error.dev.html` and `error.prod.html`